types/ckeditor__ckeditor5-autosave/ckeditor__ckeditor5-autosave-tests.ts

```diff
@@ -1,19 +1,22 @@
-import Autosave from "@ckeditor/ckeditor5-autosave";
-import { AutosaveAdapter, AutosaveConfig } from "@ckeditor/ckeditor5-autosave/src/autosave";
-import { Editor } from "@ckeditor/ckeditor5-core";
+import { Autosave } from '@ckeditor/ckeditor5-autosave';
+import { AutosaveAdapter, AutosaveConfig } from '@ckeditor/ckeditor5-autosave/src/autosave';
+import { Editor } from '@ckeditor/ckeditor5-core';
 
 class MyEditor extends Editor {}
-const plugin = new Autosave.Autosave(new MyEditor());
-plugin.once("click", () => {});
-const isContext: boolean = Autosave.Autosave.isContextPlugin;
-const state: "synchronized" | "waiting" | "saving" = plugin.state;
+const bool = true;
+const plugin = new Autosave(new MyEditor());
+plugin.once('click', () => {});
+Autosave.isContextPlugin === bool;
+const state: 'synchronized' | 'waiting' | 'saving' = 'saving';
+plugin.state === state;
 plugin.adapter.save(new MyEditor());
 
 const adapter: AutosaveAdapter = {
     save(editor: Editor) {
         return Promise.resolve(editor);
     },
 };
+adapter === adapter;
```

Comment:
what's the purpose of this? (and next one)

Reply:
It was to remove the warning about unused variables, but I finally found an use for that variable.